internal/ignition/ignition.go

```diff
@@ -205,7 +205,16 @@ func (g *installerGenerator) UploadToS3(ctx context.Context) error {
 	return uploadToS3(ctx, g.workDir, g.cluster, g.s3Client, g.log)
 }
 func (g *installerGenerator) checkLsoEnabled() bool {
```

Comment:
why isn't it part of the operator api? `IsEnabled` or even better, why not just call to generate and if it's not enabled then return an empty list or files?

Reply:
we will do it in the second part.
We decided to split it into 2 parts: 
1. swagger changes for saving the operators 
2. add the generic interface - I'll do it after Prianka will merge her code